### DISCUSSION TITLE
Add option to replace attributes in customAttributes method

### DISCUSSION THREAD

**Discussion by savosik** [moonshine-software/moonshine#1240] [Ideas]
Currently, the `customAttributes` method merges new attributes with existing ones. While this is useful in many cases, there are scenarios where we want to completely replace an attribute rather than merge it.

Proposed change:

Add an optional parameter to the `customAttributes` method to allow replacing attributes instead of merging  

```php
$component->customAttributes(['class' => 'overwitting-class'], overwrite: true);

**Comment by lee-to** [comment]
use setAttribute(name, value)
